Compile warnings on Swift 3.1+

I'm experincing some warnings when running SwiftMetrics on Swift 3.1 or 3.1.1
There are quite a few, but they are easy to fix. Here is a snippet of the logs:

```
/root/taylor/.build/checkouts/SwiftMetrics.git-8417478658542232501/Sources/SwiftMetricsBluemix/SwiftMetricsBluemix.swift:349:61: warning: string interpolation produces a debug description for an optional value; did you mean to make this explicit?
        Log.debug("[Auto-scaling Agent] sendMetrics:Error: \(error)")}
                                                            ^~~~~~~
/root/taylor/.build/checkouts/SwiftMetrics.git-8417478658542232501/Sources/SwiftMetricsBluemix/SwiftMetricsBluemix.swift:349:62: note: use 'String(describing:)' to silence this warning
        Log.debug("[Auto-scaling Agent] sendMetrics:Error: \(error)")}
                                                            ~^~~~~~
                                                             String(describing:  )
/root/taylor/.build/checkouts/SwiftMetrics.git-8417478658542232501/Sources/SwiftMetricsBluemix/SwiftMetricsBluemix.swift:349:62: note: provide a default value to avoid this warning
        Log.debug("[Auto-scaling Agent] sendMetrics:Error: \(error)")}
                                                            ~^~~~~~
                                                                   ?? <#default value#>
```

--- a/Sources/SwiftMetricsBluemix/SwiftMetricsBluemix.swift
+++ b/Sources/SwiftMetricsBluemix/SwiftMetricsBluemix.swift
@@ -346,7 +346,7 @@ public class SwiftMetricsBluemix {
         Log.debug("[Auto-scaling Agent] sendMetrics:Request: \(request!)")
         Log.debug("[Auto-scaling Agent] sendMetrics:Response: \(response!)")
         Log.debug("[Auto-scaling Agent] sendMetrics:Data: \(data!)")
-        Log.debug("[Auto-scaling Agent] sendMetrics:Error: \(error)")}
+        Log.debug("[Auto-scaling Agent] sendMetrics:Error: \(String(describing: error))")}
   }
 
   private func notifyStatus() {
@@ -359,8 +359,8 @@ public class SwiftMetricsBluemix {
       request, response, data, error in
         Log.debug("[Auto-scaling Agent] notifyStatus:Request: \(request!)")
         Log.debug("[Auto-scaling Agent] notifyStatus:Response: \(response!)")
-        Log.debug("[Auto-scaling Agent] notifyStatus:Data: \(data)")
-        Log.debug("[Auto-scaling Agent] notifyStatus:Error: \(error)")
+        Log.debug("[Auto-scaling Agent] notifyStatus:Data: \(String(describing: data))")
+        Log.debug("[Auto-scaling Agent] notifyStatus:Error: \(String(describing: error))")
     }
   }
 
@@ -376,9 +376,9 @@ public class SwiftMetricsBluemix {
       request, response, data, error in
         Log.debug("[Auto-scaling Agent] requestConfig:Request: \(request!)")
         Log.debug("[Auto-scaling Agent] requestConfig:Response: \(response!)")
-        Log.debug("[Auto-scaling Agent] requestConfig:Data: \(data!)")
-        Log.debug("[Auto-scaling Agent] requestConfig:Error: \(error)")
-        Log.debug("[Auto-scaling Agent] requestConfig:Body: \(String(data: data!, encoding: .utf8))")
+        Log.debug("[Auto-scaling Agent] requestConfig:Data: \(String(describing: data))")
+        Log.debug("[Auto-scaling Agent] requestConfig:Error: \(String(describing: error))")
+        Log.debug("[Auto-scaling Agent] requestConfig:Body: \(String(describing: data))")
         self.updateConfiguration(response: data!)
     }
   }

